packages/select/src/ListMenu/ListMenu.styles.ts

```diff
@@ -0,0 +1,44 @@
+import { css, cx } from '@leafygreen-ui/emotion';
+import { createUniqueClassName, Theme } from '@leafygreen-ui/lib';
+import { fontFamilies, Size } from '@leafygreen-ui/tokens';
+
+import { colorSets, sizeSets } from '../styleSets';
+
+export const popoverClassName = createUniqueClassName('select-popover');
+
+export const maxMenuHeight = 274;
+export const menuMargin = 8;
+
+export const baseMenuStyle = css`
+  position: relative;
+  text-align: left;
+  width: 100%;
+  border-radius: 3px;
+  line-height: 16px;
```

Comment:
Is there a `typeScale` token we should use here?

Reply:
no, we don't have a 16 token